live/mirage/models/organization.js

```diff
@@ -0,0 +1,4 @@
+import { Model } from 'ember-cli-mirage';
+
+export default Model.extend({
```

Comment:
Of course! But other models in same directory are correctly defined.
Maybe for more understanding it has to be done :) but not compulsory !